src/closure-externs.js

```diff
@@ -1370,3 +1370,40 @@ var onmessage = function(message) {};
  * @suppress {undefinedVars}
  */
 var ENVIRONMENT_IS_FETCH_WORKER;
+
+// Core functions that are exported from asm.js/wasm module to module scope
+
+/**
+ * @suppress {duplicate, undefinedVars}
+ */
+var stackSave;
+
+/**
+ * @suppress {duplicate, undefinedVars}
+ */
+var stackRestore;
+
+/**
+ * @suppress {duplicate, undefinedVars}
+ */
+var stackAlloc;
+
+/**
+ * @suppress {duplicate, undefinedVars}
+ */
+var _malloc;
```

Comment:
there should be nothing special about malloc and free that requires them to be here - this seems like it's trying to work around a remaining issue, that will occur with other functions?

Reply:
I think that's just by chance, though - there could be other functions not covered by the test suite that would be hit by this. I'm surprised there aren't already, though, so maybe I'm missing something. But this makes me uncomfortable - I think it would be better to wait for a proper solution for closure+metadce in this PR.